coral/docs/development-with-local-klaw.md

```diff
@@ -29,11 +29,14 @@ Please check out the [proxy README](../proxy/README.md) for more detailed inform
 3. Run `pnpm add-precommit` the first time you install the repository to set the custom directory for our pre commit hooks.
 4. Go to directory [`coral/proxy`](../../coral/proxy)
 5. Run `pnpm install` there, too
-6. Install Docker - [Get Docker](https://docs.docker.com/get-docker/)
-7. Go to to directory [`coral/proxy`](../../coral/proxy)
+6. Install and run Docker - [Get Docker](https://docs.docker.com/get-docker/)
```

Comment:
It's the second time I see this formatting `Things written - Other things written` so I assume it is an intentional "hyphen on the line line" formatting, and not a forgotten line break for a bullet point?

Reply:
Ah yes, I thought it might be something like this. To my eyes, I would expect this kind of "additional information" to be in parentheses :o
```
 Install and run Docker ([Get Docker](https://docs.docker.com/get-docker/))
 ```

Reply:
Hm. Looks weird to me because if you look at the readme in markdown format, it's a lot of parentheses 🙈  I'm usually reading this kind of documentation as markdown, but maybe that's not common?

Reply:
I think it is indeed not common to read docs in markdown hahaha! But I don't think it's a big deal, just that different languages have different punctuation conventions, and that this German convention doesn't translate super well :o We can still leave it as is, I feel it's a minute detail.

Reply:
OK, then let's leave it - the PR already has a lot of commits. 

So... if everything is ok now... we could merge 🤞  ?